gcc/testsuite/rust/execute/torture/macros3.rs

```diff
@@ -15,35 +15,35 @@ fn f() {
 
 macro_rules! invocation0 {
     (valid) => {
-        f()
+        f();
```

Comment:
are these test cases invalid? Do they require the semi colon in rustc?

Reply:
So they are not invalid, but they fail this PR's parsing scheme: Because the macro was invoked like so `invocation0(valid);`, the parser is trying to parse a statement, which fails since this is a `CallExpr` without a semicolon behind it. So we need to think about how to handle this. I've added a FIXME comment inside the `transcribe_rule` function where the token stream gets parsed, and I'll open up an issue